content/id/docs/reference/access-authn-authz/rbac.md

```diff
@@ -638,149 +639,156 @@ metadata:
 </tr>
 <tr>
 <td><b>cluster-admin</b></td>
-<td><b>system:masters</b> group</td>
-<td>Mengizinkan akses super-user access untuk melakukan berbagai aksi pada berbagai sumber daya. 
-Ketika digunakan pada <b>ClusterRoleBinding</b>, ini memberikan kendali penuh terhadap seluruh sumber daya pada klaster dan seluruh Namespace. 
-Ketika digunakan pada <b>RoleBinding</b>, ini memberikan kendali penuh terhadap setiap sumber daya pada Namespace RoleBinding, termasuk Namespace itu sendiri.</td>
+<td>Grup <b>system:masters</b></td>
+<td>Mengizinkan akses <i>super-user</i> untuk melakukan berbagai aksi pada berbagai sumber daya. 
+Ketika digunakan pada <b>ClusterRoleBinding</b>, Role ini akan memberikan kendali penuh terhadap semua sumber daya pada klaster dan seluruh Namespace. 
+Ketika digunakan pada <b>RoleBinding</b>, Role ini akan memberikan kendali penuh terhadap setiap sumber daya pada Namespace RoleBinding, termasuk Namespace itu sendiri.</td>
 </tr>
 <tr>
 <td><b>admin</b></td>
-<td>None</td>
-<td>mengizinkan akses admin, yang dimaksudkan untuk diberikan dalam sebuah Namespace menggunakan <b>RoleBinding</b>.
+<td>Tidak ada</td>
+<td>mengizinkan akses administrator, yang dimaksudkan untuk diberikan dalam sebuah Namespace menggunakan <b>RoleBinding</b>.
 Jika digunakan dalam <b>RoleBinding</b>, ini memungkikan akses baca/tulis ke sebagian besar sumber daya di sebuah Namespace,
 termasuk kemampuan untuk membuat Role dan RoleBinding dalam Namespace. 
 Role ini tidak memungkinkan akses tulis pada kuota sumber daya atau ke Namespace itu sendiri.</td>
 </tr>
 <tr>
 <td><b>edit</b></td>
-<td>None</td>
+<td>Tidak ada</td>
 <td>Mengizinkan akses baca/tulis pada seluruh objek dalam Namespace.
 
-Role ini tidak memungkinkan untuk melihat dan merubah Role dan RoleBinding.
+Role ini tidak memungkinkan untuk melihat dan mengubah Role dan RoleBinding.
 Namun, Role ini memungkinkan untuk mengakses Secret dan menjalankan Pod seperti ServiceAccount dalam Namespace, 
 sehingga dapat digunakan untuk mendapatkan tingkat akses API dari setiap ServiceAccount di Namespace. 
 </td>
 </tr>
 <tr>
 <td><b>view</b></td>
-<td>None</td>
+<td>Tidak ada</td>
 <td>Mengizinkan akses baca untuk melihat hampir seluruh objek dalam Namespace.
 
 Ini tidak memungkinkan untuk melihat Role dan RoleBinding.
 
 Role ini tidak memungkikan melihat Secret, karena pembacaan konten Secret memungkinkan
 akses ke kredensial ServiceAccount dalam Namespace, yang akan memungkinkan akses API sebagai
-ServiceAccount apapun di Namespace (bentuk eskalasi hak istimewa).
+ServiceAccount apapun di Namespace (bentuk eskalasi privilese).
```

Comment:
nice, I learned another new word from you https://kbbi.kemdikbud.go.id/entri/eskalasi